runtime/doc/luaref.txt

```diff
@@ -4811,7 +4811,7 @@ debug.setupvalue({func}, {up}, {value})                     *debug.setupvalue()*
         upvalue with the given index. Otherwise, it returns the name of the
         upvalue.
 
-debug.traceback([{thread},] [{message}] [,{level}])          *debug.traceback()*
+debug.traceback([{thread},] [{message} [,{level}]])          *debug.traceback()*
```

Comment:
I guess you got it from: https://www.lua.org/manual/5.1/manual.html#pdf-debug.traceback

Reply:
Yes, and my understanding is that the copy that ships with Neovim is based on that HTML source as well.

Reply:
History: #19663